### PR TITLE
Add LiveDateline component for liveblogs/deadblogs

### PR DESCRIPTION
**Pull request by iainjchambers-guardian**
## What does this change?
Adds a relative time to the header of liveblogs or deadblogs, showing time since last update

### Before
image

### After
image

### CODE REVIEW

File: apps-rendering/src/components/liveDateline.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// ----- Component ----- //
+
+const livePulse = keyframes`
+    0% {opacity: 1;}
+    10% {opacity: .25;}
+    40% {opacity: 1;}
+    100% {opacity: 1;}
+`;
+
+const timestampStyles = (isDeadBlog = false): ReturnType<typeof css> => css`
```
Comment (MarSavar) [inline apps-rendering/src/components/liveDateline.tsx:21]: I think this function can return `SerializedStyles`

**Review by MarSavar — APPROVED** [review]
LGTM!  🥳 Just one comment.

Also could you please add before/after screenshots in the PR text? 🙏

**Comment by AshCorr** [comment]
Theres a similar PR open for DCR to add this, I wonder if any of it can be re-used for AR?

https://github.com/guardian/dotcom-rendering/pull/3988

File: apps-rendering/src/components/liveDateline.tsx (hunk 1 above)
Comment (JamieB-gu) [reply apps-rendering/src/components/liveDateline.tsx:21]: What's the concern behind having an additional import? Is it a code quality/style/modularisation thing, or is it a bundle-size thing?

File: apps-rendering/src/components/liveblogHeader.tsx [hunk 2]
```diff
@@ -46,9 +45,12 @@ const LiveblogHeader: FC<Props> = ({ item }) => {
 			>
 				<Columns collapseUntil="desktop">
 					<Column span={3}>
-						<time css={timestampStyles}>
-							TODO: Updated timestamp
-						</time>
+						<Hide below="desktop">
+							<LiveDateline
+								date={item.publishDate}
```
Comment (JamieB-gu) [inline apps-rendering/src/components/liveblogHeader.tsx:50]: I think we're going to want to use some version of the "last modified" date for this. `publishDate` is when the piece was first published, which is likely to be some hours before it was actually last updated.

We don't currently have this on `Item` so I think we have two options:

1. Duplicate the logic that @AshCorr has used in #3988 to pull last modified from the most recent live block.
2. Parse `content.fields.lastModified` into a new `lastModifiedDate` on `Item` (`item.ts`) and use that.

**Review by JamieB-gu — COMMENTED** [review]
Looks good, just one point about which date to use.

> Theres a similar PR open for DCR to add this, I wonder if any of it can be re-used for AR?

I agree with you @AshCorr, duplicating this logic/UI is not ideal - I think this is exemplified by the "last modified"/"first published" question below. As discussed offline I think we'll probably merge this as is and then look into a follow-up to share some of what you've done. Perhaps we can pull the side-effect that currently sits in `PulsingDot` into a wrapper?

File: apps-rendering/src/components/liveblogHeader.tsx (hunk 2 above)
Comment (iainjchambers-guardian) [reply apps-rendering/src/components/liveblogHeader.tsx:50]: I did look at this, but the `publishDate` does get updated when the content does and using it keeps it in sync locally with the live version

Comment (JamieB-gu) [reply apps-rendering/src/components/liveblogHeader.tsx:50]: As discussed offline, I think you're right. `publishDate` is _not_ first published as I originally thought, it's actually the `webPublicationDate`, which is slightly different. For anyone else reading this, we're looking into how exactly that field gets updated, but for now I think we're ok to merge.

**Pull request merged**